<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">RandPicker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLabel" name="titleLabel">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#63b8ff;&quot;&gt;R&lt;/span&gt;andPicker&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="centerContainer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="containerLayout">
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="nameLabel">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QWidget" name="controlContainer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>60</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="controlLayout">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="settingsContainer" native="true">
         <layout class="QVBoxLayout" name="settingsLayout">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QSpinBox" name="countSpin">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="prefix">
             <string>Number: </string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="nameListCombo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Select a list</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pickButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="optionContainer" native="true">
         <layout class="QVBoxLayout" name="optionLayout">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="instantModeRadio">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Instant Pick</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="topmostRadio">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Topmost</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFeatures">
    <property name="contextMenuPolicy">
     <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
    </property>
    <property name="title">
     <string>Features(&amp;F)</string>
    </property>
    <addaction name="actionNameManager"/>
    <addaction name="separator"/>
    <addaction name="actionSidebar"/>
    <addaction name="actionSidebarToLeft"/>
    <addaction name="actionHidetoTray"/>
    <addaction name="actionRandMirage"/>
    <addaction name="separator"/>
    <addaction name="actionRPWeb"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionAboutApp"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuInPick">
    <property name="title">
     <string>InPick(&amp;I)</string>
    </property>
    <addaction name="actionScheduledPick"/>
    <addaction name="actionHistory"/>
    <addaction name="separator"/>
    <addaction name="actionImportTemp"/>
    <addaction name="actionClearPicked"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings(&amp;S)</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionFollowSystem"/>
     <addaction name="actionenUS"/>
     <addaction name="actionzhCN"/>
    </widget>
    <widget class="QMenu" name="menuRandom_Impl">
     <property name="title">
      <string>Random Implementation</string>
     </property>
    </widget>
    <addaction name="actionGlobalTracking"/>
    <addaction name="actionDisableParallelPick"/>
    <addaction name="separator"/>
    <addaction name="menuRandom_Impl"/>
    <addaction name="actionVisualEditor"/>
    <addaction name="separator"/>
    <addaction name="menuLanguage"/>
   </widget>
   <addaction name="menuFeatures"/>
   <addaction name="menuInPick"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNameManager">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::AddressBookNew"/>
   </property>
   <property name="text">
    <string>Open Namelist Manager</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionAboutApp">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DialogInformation"/>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionFollowSystem">
   <property name="text">
    <string>Follow System</string>
   </property>
  </action>
  <action name="actionenUS">
   <property name="text">
    <string notr="true">en-US</string>
   </property>
  </action>
  <action name="actionzhCN">
   <property name="text">
    <string notr="true">zh-CN</string>
   </property>
  </action>
  <action name="actionSidebar">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaSkipForward"/>
   </property>
   <property name="text">
    <string>RPSideShortcut</string>
   </property>
   <property name="toolTip">
    <string>Sidebar</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionHistory">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::SystemSearch"/>
   </property>
   <property name="text">
    <string>History</string>
   </property>
  </action>
  <action name="actionGlobalTracking">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Global Drag Tracking</string>
   </property>
  </action>
  <action name="actionClearPicked">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditClear"/>
   </property>
   <property name="text">
    <string>Clear Picked Names</string>
   </property>
  </action>
  <action name="actionScheduledPick">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentOpenRecent"/>
   </property>
   <property name="text">
    <string>Scheduled Pick</string>
   </property>
  </action>
  <action name="actionRPWeb">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::NetworkWired"/>
   </property>
   <property name="text">
    <string>RP WebSocket</string>
   </property>
  </action>
  <action name="actionHidetoTray">
   <property name="text">
    <string>Hide to Tray</string>
   </property>
  </action>
  <action name="actionImportTemp">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
   </property>
   <property name="text">
    <string>Import Temporary List</string>
   </property>
  </action>
  <action name="actionDisableParallelPick">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Disable Parallel Pick</string>
   </property>
  </action>
  <action name="actionSidebarToLeft">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaSkipBackward"/>
   </property>
   <property name="text">
    <string>RPSideShortcutLeft</string>
   </property>
  </action>
  <action name="actionVisualEditor">
   <property name="text">
    <string>Visualized Configurator</string>
   </property>
  </action>
  <action name="actionRandMirage">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::SystemLockScreen"/>
   </property>
   <property name="text">
    <string>Open RandMirage</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
